.travis.yml

```diff
@@ -13,6 +13,10 @@ matrix:
           python: 2.7
 before_install:
   - ./.travis/before_install.sh
+before_script: # configure a headless display to test plot generation
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3 # give xvfb some time to start
```

Comment:
that's weird. You seem to have done it right. Are you sure matplotlib is  not imported anywhere before?

you should also nest the imports to matplotlib in the or source tree. So matplotlib is not imported when you do `import ot`